flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/benchmark/e2e/SchedulerEndToEndBenchmarkBase.java

```diff
@@ -81,6 +87,10 @@ static DefaultScheduler createScheduler(
                 .setExecutionSlotAllocatorFactory(
                         SchedulerTestingUtils.newSlotSharingExecutionSlotAllocatorFactory(
                                 physicalSlotProvider))
+                .setRestartBackoffTimeStrategy(
```

Comment:
I prefer to add a new `createScheduler`, because other tests are not expected any failover.